gips/XXXX-optimizing-query-fees-sent-to-indexers.md

```diff
@@ -0,0 +1,64 @@
+---
+GIP: "XXXX"
```

Comment:
@howard-the-graph let's self-assign a GIP number here, 0063 is the next available

Reply:
(This is the proposed new process, which is pending official approval but some of which we've already been doing for a while: https://github.com/graphprotocol/graph-improvement-proposals/pull/30 )

Reply:
Ah, good to know. Updated ✅